Resources/config/schema/seo-1.0.xsd

```diff
@@ -12,16 +12,17 @@
             <xsd:element name="alternate-locale" type="alternate_locale" minOccurs="0" />
             <xsd:element name="error" type="error_handling" minOccurs="0" />
             <xsd:element name="sitemap" type="sitemap" minOccurs="0" />
+            <xsd:element name="content-listener" type="content_listener" minOccurs="0" />
         </xsd:sequence>
 
         <xsd:attribute name="translation-domain" type="xsd:string" />
         <xsd:attribute name="title" type="xsd:string" />
         <xsd:attribute name="description" type="xsd:string" />
         <xsd:attribute name="original-route-pattern" type="original_route_pattern" />
+        <!-- the content-key attribute should be configured in the content-listener element but this attribute is kept for BC !-->
```

Comment:
`<!-- Deprecated as of version 1.1, to be removed in version 2.0. Use the content-listener element instead.`